static/js/filter.js

```diff
@@ -0,0 +1,37 @@
+import { render } from './partials/render.js'
+
+const input = document.querySelectorAll('fieldset input')
+
+function getFilterdata () {
+  return new Promise(async (resolve, reject) => {
+    const lastValue = window.location.pathname.split('/').slice(-1)[0].split('-').slice(-1)[0]
```

Comment:
What does `.slice(-1)` do here?

Reply:
It gets the last array item that is the video id